docs/inputs/index.md

```diff
@@ -33,10 +33,10 @@ provided, a dropdown field will appear on the Inputs page. In contrast, a button
 | [warning](../advanced/custom_warning.md)                              | object | It is used to add the custom warning message for each of the modes of 'create', 'edit', 'config', and 'clone'. The message is displayed on the form.                                                          |
 | [inputHelperModule](./helper.md)                                      | string | A module that contains `validate_input` and `stream_events` methods. By default it is not used.                                                                                                               |
-| conf                                                                  | string | TBD                                                                                                                                                                                                           |
-| restHandlerName                                                       | string | TBD                                                                                                                                                                                                           |
-| restHandlerModule                                                     | string | TBD                                                                                                                                                                                                           |
-| restHandlerClass                                                      | string | TBD                                                                                                                                                                                                           |
+| conf                                                                  | string | Configuration name for a rest handler.                                                                                                                                                                        |
+| restHandlerName                                                       | string | Specify rest Handler name that extends the default behaviour of the UCC-generated REST handlers. <br>(Do NOT use with restHandlerModule)                                                                      |
+| [restHandlerModule](../advanced/custom_rest_handler.md)               | string | Specify rest Module name that extends the default behaviour of the UCC-generated REST handlers. <br>(Use with restHandlerClass)                                                                               |
+| [restHandlerClass](../advanced/custom_rest_handler.md)                | string | Specify rest Class name that extends the default behaviour of the UCC-generated REST handlers. <br>(Use with restHandlerModule)                                                                               |
```

Comment:
`restHandlerModule` and `restHandlerClass` are used to be used together and `restHandlerName` is used as standalone.
Also, per the description, `restHandlerClass` doesn't extend the UCC generated REST handlers, as the developers won't know what would be the class name until they run the ucc-gen command. You can the test cases for this.

Reply:
Same for schema.json.

Reply:
Can you suggest some description for those properties? I have never used them or worked with them so for me it is just a guess after brief research.

Reply:
@hetangmodi-crest

Reply:
- restHandlerName - Specify REST handler script name that provides fields, models and validators for the fields supported under the specified input and any specific actions (if any and as applicable) to be performed on CRUD operations for the given input. (Do NOT use with restHandlerModule or restHandlerClass)
- restHandlerModule - Specify REST handler script name that implements the custom actions (if any and as applicable) to be performed on CRUD operations for the given input. (Use with restHandlerClass)
- restHandlerClass - Specify the class name present in `restHandlerModule` implements the methods like `handleCreate`, `handleEdit`, `handleList`, `handleDelete` and is child class of `splunktaucclib.rest_handler.admin_external.AdminExternalHandler`. (Use with restHandlerModule)

Reply:
changed beginning to match to general structure and removed (if any and as applicable) as it seems to bring more confusion for me, let me know if thats fine with you and if yes we can merge it.